gravatar-ui/src/main/java/com/gravatar/ui/components/MiniProfileCard.kt

```diff
@@ -0,0 +1,75 @@
+package com.gravatar.ui.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ProvideTextStyle
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.gravatar.api.models.UserProfile
+import com.gravatar.ui.components.atomic.Avatar
+import com.gravatar.ui.components.atomic.DisplayName
+import com.gravatar.ui.components.atomic.Location
+import com.gravatar.ui.components.atomic.ViewProfileButton
+
+@Composable
+fun MiniProfileCard(profile: UserProfile, modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.Top,
+    ) {
+        Row {
```

Comment:
⚠️  We don't need the column or the horizontal/vertical alignments.

```suggestion
    Row(modifier = modifier) {
```

Reply:
Updated in ef36d1370ffed46828e611176de890930d7497d1